### ISSUE TITLE
Get full assays data

### ISSUE THREAD

**Issue by bjonnh** [mcs07/PubChemPy#10]
Is there a way to get a full assay data without having to do a manual request?

**Comment by llazzaro** [comment]
yes, you can download the json (or other format) from the pubchem offical ftp.
check here for more information https://pubchem.ncbi.nlm.nih.gov/pc_fetch/pc_fetch-help.html

**Comment by bjonnh** [comment]
Thanks @llazzaro, I was just wondering if there was a way directly from PubChemPy.

**Comment by llazzaro** [comment]
@bjonnh I understand a different thing about *full assay data*
